dev-java/jetty-util/jetty-util-9.3.5.20151012.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+
+JAVA_PKG_USE="doc source"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Jetty's Utils"
+
+MY_PN="jetty"
+MY_PV="${PV/2015/v2015}"
+MY_P="${MY_PN}-${MY_PV}"
+
+SLOT="$(get_version_component_range 1-2)"
```

Comment:
As stated in IRC that is completely wrong, as it is policy to never inherit java-utils-2.eclass. The inherits in that eclass are not for other packages, but it is used by that eclass directly. The chance of that changing in the future or ever is very little to none. Thus saying not to use things brought in by the very base java eclass in java ebuilds does not make sense what so ever. These are not things that are subject to change on a whim. If I am wrong then every Java ebuild should start inheriting java-utils-2, as that goes directly in line with your comments. We cannot rely that other java eclasses will always depend on that one. Yet even the eclass itself says not to use it directly....